platform/iphone/gl_view.mm

```diff
@@ -263,57 +88,60 @@ + (Class)layerClass {
 
 //The GL view is stored in the nib file. When it's unarchived it's sent -initWithCoder:
 - (id)initWithCoder:(NSCoder *)coder {
-	active = FALSE;
-	if ((self = [super initWithCoder:coder])) {
-		self = [self initGLES];
+	if (self = [super initWithCoder:coder]) {
+		[self setUp];
 	}
 	return self;
 }
 
-- (id)initGLES {
+- (id)initWithFrame:(CGRect)frame {
+	if (self = [super initWithFrame:frame]) {
+		[self setUp];
+	}
+	return self;
+}
+
+- (void)setUp {
+	_instance = self;
```

Comment:
Combined both initializers to this common init method. `initWithCoder:` will be called if developers create a `UIStoryboard`. If an interface is created through code, `initWithFrame:` will be called instead. Both paths require the same initialization steps.